### PR TITLE
Fix generating the redirect url

### PR DESCRIPTION
**Pull request by juliushaertl**
Fix https://github.com/nextcloud/contacts/issues/1852

### CODE REVIEW

File: lib/Controller/ContactsController.php [hunk 1]
```diff
@@ -56,7 +56,9 @@ public function __construct(IRequest $request,
 	 * @param string $uuid
 	 */
 	public function direct(string $contact): RedirectResponse {
-		$url = $this->urlGenerator->getAbsoluteURL('/apps/contacts/' . $this->l10n->t('All contacts') . '/' . $contact);
+		$url = $this->urlGenerator->getAbsoluteURL(
+			$this->urlGenerator->linkToRoute('contacts.page.index') . $this->l10n->t('All contacts') . '/' . $contact
+		);
```
Comment (skjnldsv) [inline lib/Controller/ContactsController.php:61]: No please, we have a dedicated route for this exact purpose.
So I can change it from contacts. I don't want to be server-dependant.

This looks more like a hack than a fix

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: lib/Controller/ContactsController.php (hunk 1 above)
Comment (skjnldsv) [reply lib/Controller/ContactsController.php:61]: Ah, Am I reading from right to left today??
I swear I was convinced you went from route to absolute only! :see_no_evil: :roll_eyes: 

LGTM

**Review by skjnldsv — APPROVED** [review]

**Comment by juliushaertl** [comment]
Is #nextcloud-dev still a thing :thinking: ?

**Comment by skjnldsv** [comment]
Guess not

**Pull request merged**